packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx

```diff
@@ -23,6 +24,7 @@ type UserDashboardProps = {
 
 const UserDashboard = (props: UserDashboardProps) => {
   const { type } = props;
+  const {isLoggedIn} = useUserLoggedIn();
```

Comment:
Not linted / formatted properly? Should have spaces around the object destructuring.